viral/correct_2p_sessions.py

```
@@ -373,3 +436,29 @@ def jb033_2025_06_17(c: SessionCorrection) -> SessionCorrection:
         frame_times_daq=c.frame_times_daq,
         offset_after_pre_epoch=0,
     )
+
+
+@register_correction("JB034", "2025-07-04")
+def jb033_2025_06_17(c: SessionCorrection) -> SessionCorrection:
```

Ruff (0.13.3)

Redefinition of unused `jb033_2025_06_17` from line 423

(F811)